src/run.ts

```diff
@@ -28,7 +28,8 @@ export async function run() {
       const resourceGroupName = core.getInput('resource-group', {
          required: true
       })
-      const clusterName = core.getInput('cluster-name', {required: true})
```

Comment:
File mode for both these files has changed from 0644 to 0755 - please revert that change.